packages/react-examples/src/react-charting/VerticalStackedBarChart/VerticalStackedBarChart.Styled.Example.tsx

```diff
@@ -84,7 +84,7 @@ export class VerticalStackedBarChartStyledExample extends React.Component<{}, IV
       return {
         xAxis: {
           selectors: {
-            text: { fill: 'black', fontSize: '10px' },
+            text: { fill: '#e3008c', fontSize: '10px' },
```

Comment:
Instead of using hex code use getColorFromToken how it is being used in other places in this file.

Reply:
done